app/src/test/java/mollie/tictactoe/StateConverterTest.java

```diff
@@ -0,0 +1,108 @@
+package mollie.tictactoe;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import mollie.tictactoe.players.MobilePlayer;
+import ttt.Player;
+import ttt.game.Board;
+import ttt.game.GameEngine;
+import ttt.game.Marks;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertTrue;
+
+public class StateConverterTest {
+
+    private Board mBoard = new Board(3);
```

Comment:
I don't know if it's possible with Java to create an static method that will create the instance of you object.
Get as result private Board mBoard = new Board.createBoardSize(3)

``` Java
public class Board {
  private Board() {}
  public static Board createBoardSize(int size) {
    Board board = new Board();
    // stuff
    return board;
  }
}
```